File: estudos/beecrowd/python/bee1117.py
c = soma = media = 0
while True:
    if c == 2:
        break    
    nota = float(input())
    if nota < 0 or nota > 10:
        print('nota invalida')
    else:
        soma += nota
        c += 1
media = soma / 2
print(f'media = {media:.2f}')

File: beecrowd/c/bee1117.c
#include <stdio.h>
 
int main() {
    int cont=0;
    double nota, soma=0, media;
    while(cont<2){
        scanf("%lf", &nota);
        if(nota<0 || nota>10){
            printf("nota invalida\n");
        } else {
            soma+=nota;
            cont++;
        }
    }
    media = soma/cont;
    printf("media = %.2lf\n", media);
    return 0;
}